Sentry.xcodeproj/project.pbxproj

```diff
@@ -5219,10 +5230,11 @@
 				GCC_C_LANGUAGE_STANDARD = "compiler-default";
 				GCC_OPTIMIZATION_LEVEL = 0;
 				GCC_WARN_SHADOW = YES;
-				INFOPLIST_FILE = Sources/Sentry/Info.plist;
+				INFOPLIST_FILE = Sources/Resources/Info.plist;
```

Comment:
Same thing here as with [here](https://github.com/getsentry/sentry-cocoa/pull/3206/files#r1339232979). With this one, the info plist was moved to a different path, but the old path is still reflected in the file navigator in Xcode.